composer.json

```diff
@@ -39,20 +39,21 @@
         "mremi/url-shortener": "~2.2",
         "nesbot/carbon": "^2.19",
         "phpspec/php-diff": "^1.1",
-        "sentry/sentry-laravel": "~1.0",
+        "sentry/sentry-laravel": "~4.1",
         "spatie/laravel-google-calendar": "^3.1",
+        "symfony/http-client": "*",
+        "symfony/mailgun-mailer": "*",
```

Comment:
@fsmonter Curious about these two being that they are both `"*"` - was this automatic from something else getting ran/installed?

Reply:
Added the version. I think it was added when requiring the packages.